package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clarity-js",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
>0.2.1 [](start = 14, length = 5)

Update version in core

Reply:
Oh, that's right, you didn't update package.json somehow. Update package.json and core.ts


---
In reply to: [228675204](https://github.com/Microsoft/clarity-js/pull/108#discussion_r228675204) [](ancestors = 228675204)